### ISSUE TITLE
doesn't automatically generate C_lut.npy upon install

### ISSUE THREAD

**Issue by gcadenazzi** [endolith/complex_colormap#17]
Not sure if this is the right place to mention this but I cloned this repo and installed it and it didn't automatically generate the C_lut.npy lookup table.

I had to go into complex_colormap/complex_colormap and run generate.py

I'm not sure if this is the intended behaviour or not.

**Comment by endolith** [comment]
Yeah I think that's how it has to be done.  It's a pretty big file, right?  It could probably be generated automatically on first import or something like that.

**Comment by gcadenazzi** [comment]
ah ok - understood... yeah it generates an 8.1MB file so maybe it's best to leave it to the user to generate the colormap. Although, as you said, maybe it's a good idea to have this automatically generated the first time the module is imported.

**Comment by endolith** [comment]
Yeah, or come up with a more space-efficient way to store it in the repo: https://github.com/endolith/complex_colormap/issues/10

**Comment by endolith** [comment]
If you know how to do a post-install whatever for setup.py feel free to work on making it automatic.  I don't have any experience with such things so I'll have to learn it.
